ewms_pilot/task.py

```diff
@@ -117,4 +130,7 @@ async def process_msg_task(
     if not keep_debug_dir:
         shutil.rmtree(staging_subdir)  # rm -r
 
+    _dump_binary_file(stdoutfile, sys.stdout)
+    _dump_binary_file(stderrfile, sys.stderr)
```

Comment:
I'd assume you need to do this before you delete the subdir?  And yes, this should be behind a config knob